docs/authentication.md

```diff
@@ -116,13 +65,13 @@ To regenerate your authentication token, open the API tab in your Account menu a
 
 ![regenerate-token.png](./_images/regenerate-token.png)
 
-### Saving your token as an environment variable
+## Saving your token as an environment variable
 
 If you don't want to enter a password or token each time, you can save the token as an environment variable. In some of the sample code, it is assumed that you have the token saved to your environment variables as `GROAPI_TOKEN`.
 
 Please consult your OS or IDE documentation for the most accurate and up-to-date information on how to set environment variables, e.g. setting environment variables in [Windows Powershell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-6) or [Mac OS X/Linux](https://apple.stackexchange.com/questions/106778/how-do-i-set-environment-variables-on-os-x) or [Anaconda](https://anaconda-project.readthedocs.io/en/latest/user-guide/tasks/work-with-variables.html). As a quick guide, the following steps should work:
 
-#### For Windows 10:
+### For Windows 10:

```

Comment:
These instructions don't work for Anaconda, which is our "recommended" installation method on Windows. Have the link to the Anaconda page above, but having this here seems to conflict that.

Outside the scope of this PR, but putting a note here so I don't forget: need to do a follow-up to fix that. Also the numbering is messed up in sphinx because of the newlines

Reply:
Updated